ros-conversion/cpp_message/CMakeLists.txt

```diff
@@ -58,37 +63,43 @@ ament_auto_add_library(${node_lib} SHARED
         )
 
 
-add_library(testlib SHARED IMPORTED)
+ament_target_dependencies(${node_lib}
+        rclcpp
+        rclcpp_components
+        class_loader
+        )
 
-# Following is asn1c library install path outside the container, which is usually fixed as /opt/carma/install/
-set(ASN1C_INSTALL_PATH ${asn1c_DIR}/../../../)
 
-set_target_properties(testlib PROPERTIES
-  IMPORTED_LOCATION ${ASN1C_INSTALL_PATH}/lib/libasn1c.so
-)
+target_link_libraries(${node_lib} ::stol-j2735)
+target_include_directories(${node_lib}
+        PUBLIC
+                $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+                $<INSTALL_INTERFACE:include>
+                /opt/carma/include/stol-j2735
+                )
 
-target_link_libraries(${node_lib} testlib)
-
-ament_auto_add_executable(${node_exec} 
-        src/main.cpp 
+ament_auto_add_executable(${node_exec}
+        src/main.cpp
 )
 
 # Register component
 rclcpp_components_register_nodes(${node_lib} "cpp_message::Node")
 
 # All locally created targets will need to be manually linked
 # ament auto will handle linking of external dependencies
-target_link_libraries(${node_exec}
-        ${node_lib}
-)
-
-# Testing
-if(BUILD_TESTING)  
+target_link_libraries(${node_exec} ${node_lib})
+target_include_directories(${node_exec}
```

Comment:
Seems like we have target_link_libraries and target_include_directories specified twice and this second instance removes reference to stol-j2735